plugin-locationlayer/src/main/java/com/mapbox/mapboxsdk/plugins/locationlayer/LocationLayer.java

```diff
@@ -291,13 +290,6 @@ private void styleForegroundGPS(boolean isStale) {
       isStale);
   }
 
-  void updateForegroundOffset(double tilt) {
```

Comment:
was this unused? or made obsolete?

Reply:
Obsolete, probably fixed in the Maps SDK core with `6.0.0`.